<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMConvert</class>
 <widget class="QMainWindow" name="MMConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MMConvert</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../storage/RomoShare/Movies/icons8-film-reel-48.png</normaloff>../../../../storage/RomoShare/Movies/icons8-film-reel-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbQueue">
      <property name="title">
       <string>MM Queue</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnProcess">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tvDB">
         <property name="minimumSize">
          <size>
           <width>1021</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblStatus">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMMConvert">
    <property name="title">
     <string>MMConvert</string>
    </property>
    <addaction name="actionFind_Dupes"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_MegaBoom"/>
    <addaction name="actionCompare"/>
    <addaction name="separator"/>
    <addaction name="actionMM_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionPlex_DB_Scan"/>
   </widget>
   <addaction name="menuMMConvert"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFind_Dupes">
   <property name="text">
    <string>Find Dupes</string>
   </property>
  </action>
  <action name="actionLoad_MegaBoom">
   <property name="text">
    <string>Load MegaBoom</string>
   </property>
  </action>
  <action name="actionCompare">
   <property name="text">
    <string>Compare</string>
   </property>
  </action>
  <action name="actionMM_Settings">
   <property name="text">
    <string>MM Settings</string>
   </property>
  </action>
  <action name="actionPlex_DB_Scan">
   <property name="text">
    <string>Plex DB Scan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
